extensions/wikia/Forum/Forum.i18n.php

```diff
@@ -97,46 +97,6 @@
 
 	'forum-notification-newmsg-on-followed-wall' => '$1 {{GENDER:$1|left}} a new message on the $2 board',
 
-	/* Mail message */
-	'forum-mail-notification-new-someone' => '$AUTHOR_NAME wrote a new thread on $WIKI\'s $BOARDNAME board.',
```

Comment:
Same as Wall.i18.php, there's still a bunch of these messages left. Should they be removed?
```bash
$ ls Forum.i18n.php | ack --files-from=- -c mail-notification
Forum.i18n.php:245
```